packages/contracts/src/Multisig.sol

```diff
@@ -170,7 +181,8 @@ contract Multisig is
         _setTargetConfig(_targetConfig);
     }
 
-    /// @notice Reinitializes the TokenVoting after an upgrade from a previous protocol version.
+    /// @notice Reinitializes the TokenVoting after an upgrade from a previous protocol version. For each reinitialization step, use the `_fromBuild` version to decide which internal functions to call for reinitialization.
+    /// @dev WARNING: The contract should only be upgradeable through PSP to ensure that _fromBuild is not incorrectly passed, and that the appropriate permissions for the upgrade are properly configured.
```

Comment:
I would add a description of what could happen if called with an incorrect `_fromBuild`. 
Ex, 
Calling it with incorrect `_fromBuild` could initialize already initialized values, be cautious.